### ISSUE TITLE
Search scrolls when there are multiple words

### ISSUE THREAD

**Issue by irevoire** [meilisearch/documentation#2401]
If I search for « post search » in our search bar, it suddenly scrolls a little bit, hiding the best results
![image](https://user-images.githubusercontent.com/7032172/235747607-eba47e1c-ad14-4f18-abcc-e5cc55317516.png)

I can reproduce the issue of Firefox and Chrome without any plugins.
I'm running the latest version of both on archlinux.

**Comment by maryamsulemani97** [comment]
Thanks for the issue, @irevoire! I can confirm it happens on smaller screens. I was able to reproduce it on a 13" MacBook but not on a larger screen

**Comment by dichotommy** [comment]
I don't think this error is triggered by multiple words in the query, but I'm not sure exactly what the condition is. It happens for me when I type the word "search" or the word "setting".
